src/git/Repository.ts

```diff
@@ -86,6 +86,12 @@ export class Repository {
                                 .then(() => {
                                     resolve(newRepo);
                                 });
+                        } else if (repoProperties.commit) {
+                            Global.isVerbose() && console.log(`[${repoName}]:`, 'will update to the commit', repoProperties.commit);
+                            newRepo.checkoutCommit(repoProperties.commit)
+                                .then(() => {
+                                    resolve(newRepo);
+                                });
```

Comment:
If there is a tag and commit, this block will be skipped. Shouldn't the commit be checked before the tag?

Reply:
no, I think the tag must have precedence. as the help already says:
>  1. If a tag is configured for the parent repository it is cloned on that tag,
    
> 2. Else if a commit hash is configured, the repository is cloned to the HEAD of the branch. The specific commit needs to be checked
                   out with the cplace-cli update command. --depth is ignored in that case.
                3. Else if on a release branch  ...